polaris/evaluate/_results.py

```diff
@@ -40,28 +41,24 @@ class ResultRecords(BaseModel):
 
     test_set: TestLabelType
     target_label: TargetLabelType
-    scores: dict[Metric, float]
+    scores: list[tuple[MetricType, float]]
```

Comment:
**Comment**
Not sure this is needed if you update the `Metric` class as I suggested.

Reply:
If it's a dict, the `Metric` is by default serialized to an inconvenient string that we can't directly deserialize, e.g. `"label='accuracy' config=None kind='default'"`. 

This raises a point I've run into before: It would be nice if we could have a unique (human-readable?) string identifier for each metric _within the context of a benchmark_, like we had before.